hammer.sh

```diff
@@ -552,6 +584,8 @@ function install_deps_AppImageKit()
     cd $DEPS_BUILD/AppImageKit
     curl -OL https://raw.github.com/probonopd/AppImageKit/master/AppImageAssistant.AppDir/package
     curl -OL https://raw.github.com/probonopd/AppImageKit/master/AppImageAssistant.AppDir/xdgappdir.py
+    #AppImageKit has trouble finding Ember's icon. Let's make this simpler.
+    sed -i 's/icon_name, 48, None, format/icon_name, 48/' xdgappdir.py
```

Comment:
It's because AppImageKit changed how/where it looked for the icon file. However, I fixed the issue in a better way elsewhere and I just verified that this tweak is no longer necessary. I removed it.